test/service_stubs/heroku_api_stub.rb

```diff
@@ -33,21 +33,25 @@ def two_factor?
     @body = MultiJson.decode(request.body.read) rescue {}
   end
 
-  post "/auth/reset_password" do
+  post "/password-resets" do
     MultiJson.encode({
-      message: <<-eos
-Check your inbox for the next steps.
-If you don't receive an email, and it's not in your spam folder, this could mean you signed up with a different address.
-      eos
+      created_at: Time.now.utc,
+      user: {
+        email: "kerry@heroku.com",
+        id:    "06dcaabe-f7cd-473a-aa10-df54045ff69c"
+      }
     })
+    201
   end
 
-  get "/auth/finish_reset_password/:token" do |token|
-    MultiJson.encode({ email: "kerry@heroku.com" })
-  end
-
-  post "/auth/finish_reset_password/:token" do |token|
-    MultiJson.encode({ email: "kerry@heroku.com" })
+  post "/password-resets/:token/actions/finalize" do
+    MultiJson.encode({
+      created_at: Time.now.utc,
+      user: {
+        email: "kerry@heroku.com",
+        id:    "06dcaabe-f7cd-473a-aa10-df54045ff69c"
+      }
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.